index.d.ts

```diff
@@ -0,0 +1,36 @@
+declare function Callback(error: Error, res?: any): void;
+
+declare module 'bucketclient' {
+    export class RESTClient {
+        constructor(
+            host: string | string[],
+            logApi: object | null,
+            useHttps: bool,
+            key?: string,
+            cert?: string,
+            ca?: string,
+        );
+
+        getPort(): number;
+        getRaftInformation(bucketName: string, reqUids: string[], callback: Callback, reqLogger?: any): void;
+        getBucketLeader(bucketName: string, reqUids: string[], callback: Callback, reqLogger?: any): void;
+        getBucketAttributes(bucketName: string, reqUids: string[], callback: Callback, reqLogger?: any): void;
+        putBucketAttributes(bucketName: string, reqUids: string[], attributes: string, callback: Callback, reqLogger: any): void;
+        createBucket(bucketName: string, reqUids: string[], attributes: string, callback: Callback, reqLogger: any): void;
+        deleteBucket(bucketName: string, reqUids: string[], callback: Callback, reqLogger: any): void;
+        putObject(bucketName: string, objName: string, objVal: string, reqUids: string[], callback: Callback, params: any, reqLogger: any): void;
+        getObject(bucketName: string, objName: string, reqUids: string[], callback: Callback, params, reqLogger: any): void;
```

Comment:
```suggestion
        getObject(bucketName: string, objName: string, reqUids: string[], callback: Callback, params: any, reqLogger: any): void;
```

Reply:
Same comment after the branch change. Maybe a missing commit.

Reply:
Seems to have been lost in the rebase.